src/App.scss

```diff
@@ -1,39 +1,90 @@
+* {
+  box-sizing: border-box;
+}
+
 iframe {
   display: none;
 }
 
 html {
-  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+  font-family: Cambria, Cochin, Georgia, Times, "Times New Roman", serif;
 }
 
-label,
-button {
-  cursor: pointer;
+body,
+h1,
+p {
+  margin: 0;
+}
+
+body {
+  background-color: #333;
+}
+
+.container {
+  margin: 0 auto;
+  padding: 50px 20px;
+  max-width: 1024px;
+
+  &__block {
+    margin-bottom: 20px;
+    padding: 20px;
+    background-color: #534f4f;
```

Comment:
Consider using a more descriptive class name instead of `__block`. For example, `__content` or `__section`. 🤔